Source/WebCore/crypto/SubtleCrypto.cpp

```diff
@@ -1124,7 +1124,7 @@ void SubtleCrypto::unwrapKey(JSC::JSGlobalObject& state, KeyFormat format, Buffe
     WeakPtr weakThis { *this };
     auto callback = [index, weakThis, format, importAlgorithm, unwrappedKeyAlgorithm = crossThreadCopyImportParams(*unwrappedKeyAlgorithm), extractable, keyUsagesBitmap](const Vector<uint8_t>& bytes) mutable {
         if (weakThis) {
-            if (auto promise = weakThis->m_pendingPromises.get(index)) {
+            if (RefPtr<DeferredPromise> promise = weakThis->m_pendingPromises.get(index)) {
```

Comment:
```suggestion
            if (RefPtr promise = weakThis->m_pendingPromises.get(index)) {
```

I *think* this should work fine

Reply:
just tested it and it works 👍